obspy/core/inventory/response.py

```diff
@@ -577,7 +789,7 @@ def __init__(self, stage_sequence_number, stage_gain,
                  decimation_offset=None, decimation_delay=None,
                  decimation_correction=None):
         self._symmetry = symmetry
-        self.coefficients = coefficients or []
+        self._coefficients = coefficients or []
```

Comment:
The code snippet below demonstrates a small bug in the current implementation here:
```python
from obspy.core.inventory import read_inventory
inv = read_inventory(
    'http://service.iris.edu/irisws/resp/1/query?net=GE&sta=KBS&loc=10&cha=BHZ'
    '&starttime=1999-01-01T00:00:00&endtime=2000-01-01T00:00:00')
inv.write('KBS.stationxml', format='STATIONXML')
```

This will fail with an `AttributeError` because the values `FIRResponseStage._coefficients` are not initiated as `FloatWithUncertainties`, but just as `float`. I don't fully understand what are the advantages of the implementation with `coefficients` and `_coefficients` as properties, so maybe you can comment on how the setters here work / are supposed to work @amkearns-usgs?
```python
# should be FloatWithUncertainties, but is just float:
inv.networks[0].stations[0].channels[0].response.response_stages[4].coefficients
```
It may be possible that a similar issue could also affect other Stages and their setters if the setters write directly to an attribute with `_attr` when the setter is defined for `attr`, but I have only encountered the problem right here for now.